components/python/pyzmq/Makefile

```diff
@@ -10,15 +10,32 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016 Jim Klimov
 #
 
+# See zeromq configure.ac for ABI version mapping to software version
```

Comment:
common makefiles  doesn't add much to CFLAGS/LDFLAGS besides -m$(BITS), you can check